reply-voting/app/controllers/reply_voting_controller.rb

```diff
@@ -0,0 +1,41 @@
+# @author: Aryan Ahuja
+# @created: 7/26/2021
+#
+# @project: Reply Voting
+#
+# @description: ReplyVotingController inherits from ApplicationController and listens
+#				requests made from front-end, bot POST and GET, and calls the
+#				appropriate method, depending whether it was a upvote or downvote
+
+#the file which contains various methods for update and retrieving methods from the database
+require VoteStore
+
+class ReplyVotingController < ::ApplicationController
+	
+	# precondition: coming reply id is valid
+	# postcondition: obtain a single integer representing the vote count on a specific reply
+	#
+	# @return --> vote count
+	def getVoteCount
+		VoteStore.get_count(params[:replyid])
+	end
```

Comment:
This is a really good start Aryan! However, we plan on having two counts for the total number of upvotes and the total number of downvotes. Similar to how YouTube or reddit does this. So you would need either two function to get the total upvote and downvotes. Or you could find a way to return both at the same time.

Reply:
Sure! I'll talk with the database team.

Reply:
I talked with Ryan Kwong and he created two separate methods which return counts for upvote and downvote. To avoid redundancy in the controller code, I redefined one of the routes to match the request to just one method which returns a hash, containing the count for both vote types.